samples/fpgaperf_counter/fpgaperf_counter.c

```diff
@@ -275,11 +275,11 @@ STATIC fpga_result fpga_perf_events(char* perf_sysfs_path, fpga_perf_counter *fp
 		PARSE_MATCH_INT(ptr, 0, fpga_perf->type, 10);
 
 	/* parse the format value */
-	ret = parse_perf_format(dev, fpga_perf);
+	ret = parse_perf_attributes(dev, fpga_perf, "format");
 	if (ret != FPGA_OK)
 		goto out;
 	/* parse the event value */
-	ret = parse_perf_event(dev, fpga_perf);
+	ret = parse_perf_attributes(dev, fpga_perf, "events");
```

Comment:
Should the regex pattern used the same for calling this on line 278 and 282? Shouldn't the pattern depend on the attribute kind or is only one regex pattern valid?